auth/src/main/java/org/aerogear/auth/credentials/PasswordCredentials.java

```diff
@@ -0,0 +1,23 @@
+package org.aerogear.auth.credentials;
+
+import java.util.Arrays;
+
+/**
+ * Credentials for password based authentication
+ */
+public final class PasswordCredentials implements ICredential {
```

Comment:
I'm not sure if we need a password credentials class, as we won't be handling any passwords in the app with OpenID Connect, only some tokens.